lib/controllers/app/auth_controller.dart

```diff
@@ -13,17 +14,25 @@ class AuthController extends GetxController {
 
   bool get userSignedIn => user != null;
 
-  Future<User> signInWithGoogle() async {
-    final user = await _authRepository.signInWithGoogle();
-    setUser(user);
+  // NOTE: Should only be called after user login
+  @visibleForTesting
+  Future<void> createUserDataControllerAndCreateUserEntity(User u) async {
```

Comment:
i think changing the variable name back  to user would be better for code readability

Reply:
I guess I changed it since User was already the type or something but I agree